docs/using.rst

```diff
@@ -296,7 +297,21 @@ need to issue this command in normal circumstances.
 
 ``--expand`` tells Certbot to update an existing certificate with a new
 certificate that contains all of the old domains and one or more additional
-new domains.
+new domains. With the ``--expand`` option, use the ``-d`` option to specify
+one or more domains.
+
+Example:
+
+.. code-block::
+
+  certbot --expand --cert-name existing.com -d example.com,newdomain.com
+
+If you prefer, you can specify the domains individually like this:
+
+.. code-block::
```

Comment:
When building our docs, these `code-block`s are the source of error messages and are completely absent from the built documentation. The syntax of these directives in general is described [here](http://www.sphinx-doc.org/en/stable/markup/code.html#directive-code-block), but you should make this `code-block` line and the one above:
```
.. code-block:: none
```